CHANGELOG.md

```diff
@@ -19,9 +19,129 @@ const expandWidget = new Expand({
 })
 ```
 
+### View Popup deprecations and changes
```

Comment:
Add empty line after headers for increased readability and prettier markdown syntax.